.github/workflows/deploy.yml

```diff
@@ -72,9 +72,10 @@ jobs:
           ECR_REPOSITORY: ${{ env.ecr_repository_rails }}
           IMAGE_NAME: "${{ steps.login_to_ecr.outputs.registry}}/${{ env.ecr_repository_rails }}:${{ github.sha }}"
           DOCKERFILE: ${{ env.dockerfile_rails }}
+          GEOIP_LICENSE_KEY: ${{ secrets.GEOIP_LICENSE_KEY }}
         run: |
           # Build a docker container and push it to ECR
-          docker build -t $IMAGE_NAME -f $DOCKERFILE .
+          docker build -t $IMAGE_NAME -f $DOCKERFILE --build-arg GEOIP_LICENSE_KEY --build-arg GEOIP_CACHE_BUSTER=$(date -d "`date +%Y%m01`" +%Y-%m-%d) .
```

Comment:
Instead of moving the GeoIP DB download command down in the Dockerfile, I've found (what I think is) a nicer solution: adding a cache buster build argument. What I'm doing is to passing in a build argument named `GEOIP_CACHE_BUSTER` which values is `YYYY-MM-01`, so basically the current year and month (with a fixed day). This means that this value changes every month, but that it stays the same within the same month. Once the value changes, it'll cause the cache to be invalidated and the DB is downloaded again.